src/main/java/org/broadinstitute/consent/http/service/SummaryService.java

```diff
@@ -154,204 +154,6 @@ private Summary createSummary(Integer totalPendingCases,
-                                }
-                                summaryWriter.write( StringUtils.join(dataSetUUIds, ",")  + SEPARATOR);
-                                summaryWriter.write( formatLongToDate(dar.getLong(DarConstants.SORT_DATE))  + SEPARATOR);
-                                for (User user : electionUsers){
-                                    summaryWriter.write( user.getDisplayName() + SEPARATOR);
-                                }
-                                for (int i = 0; i < (maxNumberOfDACMembers - electionUsers.size()); i++) {
-                                    summaryWriter.write(
-                                            SEPARATOR);
-                                }
-                                summaryWriter.write( booleanToString(!dar.containsKey(DarConstants.RESTRICTION))+ SEPARATOR);
-
-                                if (Objects.nonNull(chairPersonVote)) {
-                                    summaryWriter.write(booleanToString(chairPersonVote.getVote()) + SEPARATOR);
-                                    summaryWriter.write(nullToString(chairPersonVote.getRationale()) + SEPARATOR);
-                                } else {
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                }
-
-                                if (Objects.nonNull(chairPersonRPVote)) {
-                                    summaryWriter.write(booleanToString(chairPersonRPVote.getVote()) + SEPARATOR);
-                                    summaryWriter.write(nullToString(chairPersonRPVote.getRationale()) + SEPARATOR);
-                                } else {
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                }
-
-                                if (Objects.nonNull(chairPersonConsentVote)) {
-                                    summaryWriter.write(booleanToString(chairPersonConsentVote.getVote()) + SEPARATOR);
-                                    summaryWriter.write(nullToString(chairPersonConsentVote.getRationale()) + SEPARATOR);
-                                } else {
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                    summaryWriter.write(nullToString(null) + SEPARATOR);
-                                }
-                            }
-                        }
-                        summaryWriter.write(END_OF_LINE);
-                    }
-                }else file = null;
-                summaryWriter.flush();
-            }
-            return file;
-        } catch (Exception e) {
-            logger.error(e.getMessage());
-        }
-        return file;
-    }
-
```

Comment:
As fallout from this, it looks like these methods are now unused:
* setSummaryHeader
* setSummaryHeaderDataAccessRequest
* findAssociatedDAR
* booleanToString
* nullToString

Also, these finals are no longer in use:
* consentDAO
* MANUAL_REVIEW